lib/monitoring/aws-apigateway/ApiGatewayMonitoring.ts

```diff
@@ -37,16 +37,42 @@ import {
   ApiGatewayMetricFactoryProps,
 } from "./ApiGatewayMetricFactory";
 
+const DefaultLatencyTypesToRender = [
+  LatencyType.P50,
+  LatencyType.P90,
+  LatencyType.P99,
+];
+
 export interface ApiGatewayMonitoringOptions extends BaseMonitoringProps {
   readonly addLatencyP50Alarm?: Record<string, LatencyThreshold>;
```

Comment:
This doesn't seem particularly scalable. Someone was fiddling with more advanced interface/type generation based on the enum before that didn't pan out in time for them, but it may be worth looking at that again at some point.

Reply:
I know, it is not the most elegant solution. I replicated the existing solution from Coral monitoring, to at least achieve some consistency. We can iterate and improve all places once we have a better solution...